### DISCUSSION TITLE
GraphQL and Nest.js enforce implementing field resolvers?

### DISCUSSION THREAD

**Discussion by m1212e** [prisma/prisma#20038] [Q&A] [answered]
Hi,

I'm building a GraphQL API with Nest.js and Prisma and I have an entity with a few relations. I would like to somehow enforce that the relations this type has will be made available via field resolvers. Is this a good idea and how could I achieve this? Maybe implementing the prisma interface in my entity class might help?

**Comment by ludralph** [comment, marked as answer]
Hi @m1212e 👋 

You might want to look at this library [https://www.npmjs.com/package/prisma-nestjs-graphql](prisma-nestjs-graphql). It would be useful in your usecase.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
@m1212e Did you had a chance to look at Raphael's response? Please let us know in case you still need any help. If not, please consider marking the discussion as answered/closed.
